files/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/bash
+
+export RELEASE_BOT_HOME=/home/release-bot
+
+exec release-bot -c /home/release-bot/.config/conf.yaml &
+exec celery -A release_bot.celery_task worker -l info
```

Comment:
I've never seen running a "service" and celery worker(s) in the same pod/container.
It IMHO doesn't make sense to use `celery` in that case.

You most likely want to run release-bot (the Flask instance) in one pod and celery worker in another (which you can then scale up/down depending on the load)

Reply:
@jpopelka Yes, I agree that Flask instance and celery should run in two separate pods. However, at the moment, I don't have too much time to work on it. Can we create an issue for that?
The rest of the previous reviews should be fine now and the github app is working.

Reply:
Ok, I'm fine with that for now if it works. Will you create that issue, please?